metadataclient/tests/conftest.py

```diff
@@ -4,9 +4,9 @@
 import tzlocal
 import pytest
 import metadataclient.mds
-import metadataclient.commands
+# import metadataclient.commands
```

Comment:
Does this mean we are deprecating the commands api in metadatastore as well?
If so, I need to go around and check everyone's configs to make sure they're not using the commands instead of mds.MDS